cmd/limactl/start.go

```diff
@@ -276,8 +273,8 @@ func loadOrCreateInstance(cmd *cobra.Command, args []string, createOnly bool) (*
 			return nil, err
 		}
 	}
-	saveBrokenEditorBuffer := tty
-	return createInstance(cmd.Context(), st, saveBrokenEditorBuffer)
+	saveBrokenYAML := tty
+	return create.Create(cmd.Context(), st.instName, st.yBytes, saveBrokenYAML)
```

Comment:
Should we rename `Create` to `Instance`?
```suggestion
	return create.Instance(cmd.Context(), st.instName, st.yBytes, saveBrokenYAML)
```

Reply:
> Should we rename `Create` to `Instance`?

No, package names are supposed to be nouns and function names verbs, so this would be backwards. When a function name is a noun, it should be a Getter, with an implied `Get` prefix to make it a verb, so `Instance` would mean `GetInstance`.

Anyway, I think the solution is to create an `instance` package instead, so we have `instance.Create`.

It has the added benefit that the `start`, `stop`, and `delete` (does not exist yet) packages can be merged into `instance` as well, instead of having a single file in each packages. So `start.Start()` will become `instance.Start()` etc.